examples/meta/src/kernel/linear.sg.in

```diff
@@ -0,0 +1,20 @@
+File traindat = csv_file("@SHOGUN_DATA@/fm_train_real.dat")
+File testdat = csv_file("@SHOGUN_DATA@/fm_test_real.dat")
+
+#![create_features]
+Features feats_train=features(traindat)
+Features feats_test=features(testdat)
+#![create_features]
+
+#![create_kernel]
+Kernel linear_kernel=kernel("LinearKernel")
+KernelNormalizer avg_diag_normalizer = AvgDiagKernelNormalizer()
+linear_kernel.set_normalizer(avg_diag_normalizer(1.2))
+linear_kernel.init(feats_train, feats_train)
+#![create_kernel]
```

Comment:
@vigsterkr, I've made a force push, with all the corrections you've asked for. I hope you will now respond to my queries.
Running ```make && make meta_examples```  gave the following output https://pastebin.com/1cTQxeC4 

Which doesn't create any new files.

Reply:
the files will be in `build/examples/meta/` and then you'll see the different language targets :)

Reply:
It should've been the case, but Visual studio code notified no creation of files. I even checked ```build/examples/meta/python/kernel/``` and I couldn't find ```linear.py``` as was expected

Reply:
btw `make meta_examples` is enough to translate the examples, `make` just builds all targets. To compile the C++ target of your meta example it should be `make cpp-kernel-linear`

Reply:
does visual studio have it's own build folder?

Reply:
I'm using docker for the ```make meta_examples``` commands and only using VSC to know the changes occured in the folder. I've ran all the ```mkdir build, cmake, make, etc``` through my docker image only

Reply:
ok, can you see the file in your terminal?

Reply:
> btw `make meta_examples` is enough to translate the examples, `make` just builds all targets. To compile the C++ target of your meta example it should be `make cpp-kernel-linear`

Ran the command, got this
```make: *** No rule to make target 'cpp-kernel-linear'.  Stop.```

Reply:
> ok, can you see the file in your terminal?

Nope, my docker /home and host /home are the same(share volume), so any changes in docker should be visible through VSC.

Reply:
did you pass `-DENABLE_TESTING=ON  DISABLE_META_CPP=OFF` with `cmake`? For a complete list of these variables use `ccmake` please

Reply:
from your pastebin your example wasn't built actually, so run `cmake ..` again

Reply:
can you bring this conversation to irc please?